tests/unit/test_models/test_submodels/test_particle/test_size_distribution/test_fast_many_distributions.py

```diff
@@ -0,0 +1,57 @@
+#
+# Test many size distributions of particles with internal uniform profile
+#
+
+import pybamm
+import tests
+import unittest
+
+
+class TestManySizeDistributions(unittest.TestCase):
```

Comment:
getting rid of these submodel tests anyway so no need to add

Reply:
Ok, removed them (I modified the existing particle submodel tests just so that they pass, didn't bother to fully rename everything there).